### PR TITLE
Features/report

### PR DESCRIPTION
**Pull request by dorier**
Added 5 projects, and corresponding members and publications.

### CODE REVIEW

File: _projects/extreme_scale_workflows.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+layout: page_project
+title: Extreme-Scale Workflow Tools: Swift, Decaf, Damaris, FlowVR
+date: 2016-03-21
+updated:
+navbar: Research
+subnavbar: Projects
+project_url:
+status: running
+topics: 
+  -
```
Comment (torbjoernk) [inline _projects/extreme_scale_workflows.md:11]: Does this project really have none of the official seven JLESC topics associated?

File: _projects/interf_aware_sched.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+layout: page_project
+title: Towards Interference-aware scheduling in HPC systems
+date: 2016-03-21
+updated:
+navbar: Research
+subnavbar: Projects
+project_url:
+status: running
+topics: storage
+  -
```
Comment (torbjoernk) [inline _projects/interf_aware_sched.md:11]: Unfortunately, this is causing an error. Either delete the line with the single dash or put the project ID `storage` after the dash:

```
[...]
status: running
topics: storage
keywords:
[...]
```

or

```
[...]
status: running
topics:
  - storage
keywords:
[...]
```

File: _projects/omniscio.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+layout: page_project
+title: Exploiting Omnisc'IO to improve scheduling, prefetching and caching in Exascale systems
+date: 2016-03-21
+updated:
+navbar: Research
+subnavbar: Projects
+project_url:
+status: running
+topics: storage
```
Comment (torbjoernk) [inline _projects/omniscio.md:10]: same here, see above

File: _projects/scheduling_hpc_workflows.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+---
+layout: page_project
+title: Resource Management and Scheduling for Data-Intensive HPC Workflows
+date: 2016-03-21
+updated:
+navbar: Research
+subnavbar: Projects
+project_url:
+status: starting
+topics: storage
```
Comment (torbjoernk) [inline _projects/scheduling_hpc_workflows.md:10]: same here, see above

File: _projects/smart_in_situ.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+layout: page_project
+title: Smart In Situ Visualization
+date: 2016-03-21
+updated:
+navbar: Research
+subnavbar: Projects
+project_url:
+status: running
+topics: 
+  -
```
Comment (torbjoernk) [inline _projects/smart_in_situ.md:11]: Also a project without a topic?

**Comment by torbjoernk** [comment]
Thanks a lot for your contribution! There are a few minor points currently holding us back from merging this in, which can easily be fixed (and actually also be done by the one merging it in).

Do the "_Smart In Situ Visualization_" and "_Extreme-Scale Workflow Tools: Swift, Decaf, Damaris, FlowVR_" projects really have no JLESC topic? Each project can be associated with multiple topics.

However, in case they do not really fit into any of the seven topics, I have to rethink the project overview page of the website.

**Comment by dorier** [comment]
"storage" seems to be one of the topics, I don't know why it's not
recognized. As for the projects that don't have a topic, they can be put in
"storage" as well, I think.

Note that the topic was marked "optional" in the template...

Matthieu

2016-03-21 11:51 GMT-05:00 Torbjörn Klatt notifications@github.com:

> Thanks a lot for your contribution! There are a few minor points currently
> holding us back from merging this in, which can easily be fixed (and
> actually also be done by the one merging it in).
> 
> Do the "_Smart In Situ Visualization_" and "_Extreme-Scale Workflow
> Tools: Swift, Decaf, Damaris, FlowVR_" projects really have no JLESC
> topic? Each project can be associated with multiple topics.
> 
> However, in case they do not really fit into any of the seven topics, I
> have to rethink the project overview page of the website.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/JLESC/jlesc.github.io/pull/162#issuecomment-199374445

## 

---

Matthieu Dorier

**Comment by pancetta** [comment]
@dorier Thank you for submitting all this! The topic should be mandatory, we'll change that asap in the tenplate. I'll take care of the rest tomorrow.

**Comment by pancetta** [comment]
@dorier I merged your contribution, your projects are all online now. Thanks again!

**Pull request merged**